hw3.1/index.html.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style1.css">

```

Comment:
Fix style link name. And html doc name

Reply:
done

Reply:
Please also remove old file index.html.html

Reply:
+